sky/spot/recovery_strategy.py

```diff
@@ -171,8 +157,8 @@ def _launch(self, max_retry=3, raise_on_failure=True) -> Optional[float]:
                                                            get_end_time=False)
                 return launch_time
 
-            # TODO(zhwu): maybe exponential backoff is better?
-            if retry_cnt >= max_retry:
+            if max_retry and retry_cnt >= max_retry:
```

Comment:
styleguide nit: `max_retry` -> `max_retry is not None`

Reply:
Done. Thanks!